src/flat-button.jsx

```diff
@@ -265,6 +265,7 @@ const FlatButton = React.createClass({
       userSelect: 'none',
       position: 'relative',
       overflow: 'hidden',
+      verticalAlign: 'middle',
```

Comment:
@oliviertassinari Thanks.. That seems to be a good option. I tested it and it works fine.

My only thought was whether all the buttons should have a default vertical-align or not.

I'll make the changes and push to the PR now.